src/functions/keycloak/find-user/handler.ts

```diff
@@ -0,0 +1,64 @@
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { CreateKeycloakApiService, type KeycloakApiService } from "src/apis/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Response schema for lambda
+ */
+interface Response {
+  id: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  isActive: boolean;
+  severaGuid: string;
+  forecastId: number;
+}
+
+/**
+ * Gets user from keycloak
+ *
+ * @param api
+ * @param id
+ * @returns user information by Id
+ */
+const findUser = async (api: KeycloakApiService, id: string): Promise<Response[]> => {
+  return await api.findUser(id);
+};
+
+/** 
+ * Lambda for finding user
+ * 
+ * @param event event
+ * @returns user information as string
+ */
+
+const findUserHandler: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent) => {
+  let apiResponse;
```

Comment:
Is this actually needed,  can't you just return the appropriate objects without assigning them to this apiResponse (which is initialized as type any)- something we want to avoid.